spec/rack/test/cookie_spec.rb

```diff
@@ -28,6 +28,31 @@
       expect(last_request.cookies).to eq({})
     end
 
+    it 'uses the first path when many paths are defined' do
+      cookie_string = [
+        '/',
+        'csrf_id=ABC123',
+        'path=/, _github_ses=ABC123',
+        'path=/',
+        'expires=Wed, 01 Jan 2020 08:00:00 GMT',
+        'HttpOnly'
+      ].join('; ')
+      cookie = Rack::Test::Cookie.new(cookie_string)
+      expect(cookie.path).to eq('/')
+    end
+
+    it 'uses the first path when many paths are defiend' do
+      cookie_string = [
+        '/',
+        'csrf_id=ABC123',
+        'path=/',
+        'expires=Wed, 01 Jan 2020 08:00:00 GMT',
+        'HttpOnly'
+      ].join('; ')
+      cookie = Rack::Test::Cookie.new(cookie_string)
+      expect(cookie.path).to eq('/')
+    end
```

Comment:
This one still has the typo + misleading name. `it 'properly uses a single path being defined'` or something is fine.